From e769eca5eb74eb36f0aee22d920bee652af0a744 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <adrian.alonso@nxp.com>
Date: Thu, 5 Jul 2018 12:02:24 +0900
Subject: [PATCH 4075/5242] MLK-18775: ARM64: dts: freescale: imx8mm: fix
 ecspi regs

commit  ebefd8f107e439d7e73ccda7b4b42cd29ce42289 from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix incorrect reg address for ESCPI2/ECSPI3

Signed-off-by: Adrian Alonso <adrian.alonso@nxp.com>
(cherry picked from commit cf15610449eb581adb8c2f7f1bb420191b4ee617)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index 2088eb2..a395b70 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -837,7 +837,7 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mm-ecspi", "fsl,imx51-ecspi";
-		reg = <0x0 0x30820000 0x0 0x10000>;
+		reg = <0x0 0x30830000 0x0 0x10000>;
 		interrupts = <GIC_SPI 32 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8MM_CLK_ECSPI2_ROOT>,
 			<&clk IMX8MM_CLK_ECSPI2_ROOT>;
@@ -849,7 +849,7 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mm-ecspi", "fsl,imx51-ecspi";
-		reg = <0x0 0x30820000 0x0 0x10000>;
+		reg = <0x0 0x30840000 0x0 0x10000>;
 		interrupts = <GIC_SPI 33 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8MM_CLK_ECSPI3_ROOT>,
 			<&clk IMX8MM_CLK_ECSPI3_ROOT>;
-- 
1.7.9.5

